hkmc2/shared/src/test/mlscript/basics/BadModuleUses.mls

```diff
@@ -4,13 +4,20 @@
 module M with
   fun mtd() = 42
 
+:e
 module Example with
   fun foo(): module null = M
+//│ ╔══[ERROR] Expected a module, found non-moduleful null literal.
+//│ ╙── Function marked as 'module' must have a module signature.
```

Comment:
This function is not "marked as 'module'" 🤔

Reply:
How do you say that, "function signature marked as 'module'..."?

Reply:
No... it's not the signature (which includes parameter types) that's marked. You want to say
```suggestion
//│ ╙── Function marked as returning a 'module' must return a module type.
```